demisto_sdk/scripts/update_additional_dependencies.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+from pathlib import Path
+from typing import Optional, Sequence
+
+from demisto_sdk.commands.common.git_util import GitUtil
+from demisto_sdk.commands.common.handlers import YAML_Handler
+from demisto_sdk.commands.common.logger import logger
+from demisto_sdk.commands.common.tools import get_file
+
+yaml = YAML_Handler()
+
+
+def update_additional_dependencies(
+    pre_commit_config_path: Path, requirements_path: Path, hooks: Sequence[str]
+) -> int:
+    try:
+        if "poetry.lock" not in GitUtil().get_all_changed_files():
+            logger.info(
+                "Skipping update of additional dependencies since poetry.lock was not changed"
+            )
+            return 0
+        requirements = requirements_path.read_text().splitlines()
+        pre_commit = get_file(pre_commit_config_path)
+        for repo in pre_commit["repos"]:
+            for hook in repo["hooks"]:
+                if hook["id"] in hooks:
+                    hook["additional_dependencies"] = requirements
+        with pre_commit_config_path.open("w") as f:
+            yaml.dump(pre_commit, f)
+        return 0
+    except Exception as e:
+        logger.error("Failed to update additional dependencies: %s", e)
+        return 1
```

Comment:
why return 0/1 and `SystemExit` with it? isn't raising preferred?

Reply:
inspired by the hooks here: https://github.com/pre-commit/pre-commit-hooks/tree/main/pre_commit_hooks